06 - Inbox/🗂️ 06 - Inbox.md

```diff
@@ -26,7 +26,7 @@ In this folder are all new things which haven't been properly sorted into the Ob
 -  [[06 - Inbox/Productivity Guru|Productivity Guru]]
 -  [[06 - Inbox/Seedbox|Seedbox]]
 -  [[06 - Inbox/SkillShare|SkillShare]]
--  [[06 - Inbox/Yet Another Free Publish Alternative|Yet Another Free Publish Alternative]]
+-  [[Obsidian2Mkdocs|Yet Another Free Publish Alternative]]
```

Comment:
This change is harmless, so there's no need to change it... but just to note that it will get over-written as this content is generated by script.

Reply:
The link name was changed because I rename the old note !